packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -167,3 +170,16 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
     );
   },
 );
+
+const NotificationCountBadgeManageButton = () => {
+  const { membership } = useCoreOrganization();
+  const { organizationSettings } = useEnvironment();
+  const isAdmin = membership?.role === 'admin';
```

Comment:
Irrelevant to this PR, but that's a nice candidate to introduce a `HasRole()` method on the MembershipResource